samples/net/coap_client/README.rst

```diff
@@ -1,13 +1,13 @@
-.. _nrf_coap_client_sample:
+.. _coap_client_sample:
 
-nRF CoAP Client
-###############
+CoAP Client
+###########
 
 .. contents::
    :local:
    :depth: 2
 
-The nRF CoAP Client sample demonstrates the communication between a public CoAP server and an nRF91 Series SiP or nRF70 Series SoC that acts as the CoAP client.
+The CoAP Client sample demonstrates the communication between a public CoAP server and an nRF91 Series SiP or nRF70 Series SoC that acts as the CoAP client.
```

Comment:
```suggestion
The CoAP Client sample demonstrates the communication between a public CoAP server and a CoAP client application that is running on a Nordic SoC that enables IP networking via cellular or Wi-Fi connectivity. Cellular connectivity is supported on nRF91 Series SiP, while Wi-Fi connectivity is supported on nRF52 or nRF53 Series SoCs hosting nRF70 Series Wi-Fi companion ICs.
```

Reply:
@simensrostad since you're improving Docs, I took the liberty of improving the headline here, to make it more accurate for Wi-Fi at least. @richabp to review the wording :)

Reply:
@divipillai Could you address this comment when going over the docs?

Reply:
My suggestion:
The CoAP Client sample demonstrates the communication between a public CoAP server and a CoAP client application that is running on a Nordic SoC that enables IP networking via cellular or Wi-Fi connectivity.
Cellular connectivity is supported on **the** nRF91 Series SiP, while Wi-Fi connectivity is supported on **the** nRF52 or nRF53 Series SoCs hosting **the** nRF70 Series Wi-Fi companion ICs.